app/views/decidim/vocdoni/admin/answers/edit.html.erb

```diff
@@ -1,10 +1,13 @@
-<% add_decidim_page_title(translated_attribute(answer.title)) %>
 <% add_decidim_page_title(t(".title")) %>
```

Comment:
Can we leave the resource title in the page title please? So it's easier to differentiate from other resources.  Also let's have a break-line (as it was before) so it easy to have the meta and body blocks visually separated. 

```suggestion
<% add_decidim_page_title(t(".title")) %>
<% add_decidim_page_title(translated_attribute(answer.title)) %>

```

Reply:
I think these changes weren't applied, can you check it?